pkg/bootstrap/platform/k8s.go

```diff
@@ -106,8 +106,8 @@ spec:
       containers:
         - name: cl-controlplane
           image: cl-controlplane
-          imagePullPolicy: IfNotPresent
           args: ["--log-level", "info", "--platform", "k8s"]
+          imagePullPolicy: IfNotPresent
```

Comment:
nit: AFAIK, using `imagePullPolicy: Always` can be better in dev-loop where the tags aren't changed on every build (it'll continue to use the already cached/present image instead of picking up the new one.

Reply:
Personally, as a dev, I use the tests to verify my code, and not some static kind cluster.
Also, the template here should be configured for production, not dev use.